llarp/handlers/tun.cpp

```diff
@@ -258,27 +258,29 @@ namespace llarp
         return service::Address{itr->second.as_array()};
     }
 
+    constexpr auto TrafficAlignmentTimeout = 10s;
+
     bool
     TunEndpoint::HandleHookedDNSMessage(dns::Message msg, std::function<void(dns::Message)> reply)
     {
-      auto ReplyToSNodeDNSWhenReady = [self = this, reply = reply](
-                                          RouterID snode, auto msg, bool isV6) -> bool {
-        return self->EnsurePathToSNode(
+      auto ReplyToSNodeDNSWhenReady = [this, reply](RouterID snode, auto msg, bool isV6) -> bool {
+        return EnsurePathToSNode(
             snode,
-            [=](const RouterID&, exit::BaseSession_ptr s, [[maybe_unused]] service::ConvoTag tag) {
-              self->SendDNSReply(snode, s, msg, reply, isV6);
+            [this, snode, msg, reply, isV6](
+                const RouterID&, exit::BaseSession_ptr s, [[maybe_unused]] service::ConvoTag tag) {
```

Comment:
RouterID argument and ConvoTag argument are both not used; they should have the same format to declare as much.